pkg/driver/aws-ebs/aws_ebs.go

```diff
@@ -70,6 +77,9 @@ func GetAWSEBSGeneratorConfig() *generator.CSIDriverGeneratorConfig {
 					"--kube-api-burst=100",
 					"--worker-threads=100",
 				),
+				commongenerator.DefaultProvisionerWithSnapshots.WithAdditionalAssets(
+					"base/rbac/volumeattributesclass_reader_provisioner_binding.yaml",
+				),
```

Comment:
don't add a new `DefaultProvisionerWithSnapshots`, add new parameters to the one added few lines above